generators/app/templates/src/redux/store.ejs

```diff
@@ -6,7 +6,6 @@ import {
   createNavigationReducer
 } from 'react-navigation-redux-helpers';
 import { fetchMiddleware, configureMergeState } from 'redux-recompose';
-import Navigator from '@screens';
```

Comment:
Don't delete this line, you must only add a blank line between `
import { fetchMiddleware, configureMergeState } from 'redux-recompose';`  and `import Navigator from '@screens';`